src/Auth0.ManagementApi/Clients/GuardianClient.cs

```diff
@@ -119,13 +115,12 @@ public Task<GuardianTwilioConfiguration> GetTwilioConfigurationAsync()
         public Task<UpdateGuardianFactorResponse> UpdateFactorAsync(UpdateGuardianFactorRequest request)
         {
             var name = request.Factor == GuardianFactorName.PushNotifications ? "push-notification" : "sms";
+
             return Connection
-                .PutAsync<UpdateGuardianFactorResponse>(
-                "guardian/factors/{name}",
-                new { enabled = request.IsEnabled },
-                null, null,
-                new Dictionary<string, string> { { "name", name } },
-                null, null);
+                .SendAsync<UpdateGuardianFactorResponse>(
+                HttpMethod.Put,
+                BuildUri($"guardian/factors/{name}"),
+                new { enabled = request.IsEnabled });
```

Comment:
**Question:** What is `new` doing here?

Reply:
It's creating a new anonymous object.  As the put only contains a single `enabled` property inside a JSON object it was deemed simpler to pass in a boolean rather than creating a Request object for it with just one property.